### ISSUE TITLE
Scaling Rectified Flow Transformers for High-Resolution Image Synthesis

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#485]
## Links
https://arxiv.org/abs/2403.03206

### ◆3行まとめ
本論文は，Rectified Flow に基づく新しいノイズサンプリング手法と，テキスト・画像の両モダリティを学習可能に扱う Transformer アーキテクチャ MM-DiT を提案する。  
これにより，高解像度テキスト生成画像タスクにおいて，従来のDiffusion/EDM/DiT 系手法を大幅に上回る性能を達成した。  
さらに，大規模スケーリング実験により，モデルサイズと性能向上の明確な相関（スケーリング則）を示し，8Bモデルが既存のSOTAモデルを超えることを実証した。

<img width="666" height="470" alt="Image" src="https://github.com/user-attachments/assets/3289b049-0d8e-41b8-9add-797c2bbf3b85" />

---

# 構造的な概要

## 1. 研究背景
Diffusionモデルは高品質な画像生成で主流となっている一方，**推論の遅さ**や**ノイズスケジュール依存性**が課題である。Rectified Flow（RF）は「データ–ノイズ間の直線経路」を学習するため理論的に有利だが，大規模T2Iでは十分に検証されてこなかった。

本研究は以下を明確に示す：
1. **Rectified Flow を大規模T2Iで機能させる方法論の構築**  
2. **テキストと画像を双方向に統合する新アーキテクチャの提案**  
3. **スケーリング則を備えた高性能モデルの実証**

---

### (1) Rectified Flow 向けの新しい timestep サンプリング（Logit-Normal Sampler）
- 従来のRFは **Uniform sampling** を用いており，中間のノイズ領域がうまく学習されず性能が頭打ちだった。
- 本研究は **Logit-Normal / Mode / CosMap** といった重み付け分布を新たに提案し，中間領域を重点的に学習。
- 大規模比較（61変種）により **rf/lognorm(0,1)** が安定して SOTA を達成（Tab.1,2）。

### (2) 新アーキテクチャ **MM-DiT**（Multimodal DiT）
- 従来：DiT や U-Net は **テキストと画像を単方向 or 弱い統合**で扱う。
- 本研究：  
  - **テキスト・画像用に独立した重みを持つ2系列 Transformer**  
  - Self-Attention は両系列を結合し **双方向に情報交換**  
  - 図2（p.5）では token-level の情報流通構造を可視化。  
- 結果：テキスト理解・タイポグラフィ生成・位置理解が大幅に向上（Fig. 4）。

### (3) 大規模スケーリング（最大 8B パラメータ）
- 検証されたスケーリング則を示し，**モデルサイズ↑ → Validation Loss↓ → 評価指標↑** の明確な傾向（Fig. 8）。
- 8Bモデルは SDXL・Pixart-α・DALL-E 3 を含む複数のSOTAを超えた（Fig. 7, Table 5）。

---

### (1) 大規模比較（61 variants）
- ImageNet / CC12M で 61種類のRF/EDM/Linear/Cosineなどを統一条件で比較。
- CLIP score, FID, validation loss, sampling steps を横断的に評価。
- Pareto-based ranking によって **rf/lognorm(0,1)** が最も安定した性能を示すことを確認（Table 1）。

### (2) アーキテクチャ比較
DiT / CrossDiT / UViT / MM-DiT（2-weight / 3-weight）で，
- Validation loss（学習曲線）
- CLIP-score
- FID  
を比較（Fig. 4）。MM-DiT が全指標で最良。

### (3) 大規模スケーリング実験（256²〜1024²）
- 500k steps の pretraining + 高解像度 finetuning（1024²）。
- Validation loss と GenEval / T2I-CompBench / Human preference の強い相関を示す（Fig. 8）。
- 8B + DPO 調整で最終的に全 SOTA を上回るスコア（Table 5）。

---

### ● **Figure 3（p.7）** – Rectified Flow の sample efficiency  
- sampling steps を減らしても性能が維持されることを可視化。  
- 特に rf/lognorm(0,1) は 5〜10 steps でも高品質を保つ。

### ● **Figure 4（p.9）** – アーキテクチャ比較  
- Validation loss, CLIP, FID の全てで MM-DiT が DiT/UViT を上回る。  
- 特に学習安定性と収束の速さで差が顕著。

### ● **Figure 7（p.10）** – 人間評価（Parti-prompts）  
- 8Bモデル（ours）は SDXL / DALL-E 3 / Pixart などを超え，  
  - **Prompt Following**  
  - **Visual Aesthetics**  
  - **Typography**  
 で優位性を示す。

### ● **Figure 8（p.12）** – スケーリング曲線  
- 画像・動画モデルに対して「モデルサイズ↑ → validation loss↓」を示す。  
- Validation loss と外部評価指標（GenEval, T2I-CompBench, Human preference）に強い線形相関。

### ● **Table 5（p.11）** – GenEval の総合比較  
- depth=38 (8B) モデルが DALL-E 3 や SDXL Turbo を上回り、  
  特に  
  - Counting  
  - Two Objects  
  - Color Attribution  
 で最高スコア。

---

# 結論
本論文は，
1. **Rectified Flow モデルを大規模テキスト画像生成に最適化するノイズサンプラーの開発**  
2. **テキスト・画像の双方向統合を行う新アーキテクチャ MM-DiT の提案**  
3. **8B-scale での徹底的なスケーリング実験による SOTA 超えの実証**

を通じ，Rectified Flow 系モデルを Diffusion の強力な代替として確立する大きな成果を示した。
